deploy/charts/external-secrets/README.md

```diff
@@ -75,6 +75,7 @@ The command removes all the Kubernetes components associated with the chart and
 | crds.createClusterExternalSecret | bool | `true` | If true, create CRDs for Cluster External Secret. |
 | crds.createClusterSecretStore | bool | `true` | If true, create CRDs for Cluster Secret Store. |
 | createOperator | bool | `true` | Specifies whether an external secret operator deployment be created. |
+| dnsOptions | object | `{}` | Specifies `dnsOptions` to the external-secrets controller |
```

Comment:
The chart readme is autogenerated, please add comments to the `values.yaml` like below and run `make helm.docs`, that should autogenerate this file.

```yaml
## -- Map of extra arguments to pass to container.
extraArgs: {}
```

Reply:
I did not realized that thanks for the clarification let me quickly update it.